routes/api.php

```diff
@@ -39,6 +39,7 @@
     Route::get('/getLatestTask', [TaskController::class, 'getLatestTask']);
     Route::get('/todo_resource', [TodoController::class, 'showResource']);
     Route::put('add/{id}', [TaskController::class, 'addTask']);
+    Route::put('/todo/{id}/mark-task', [TaskController::class, 'markTask']);
```

Comment:
@itz-bethel, for easy consumption of API, please model the routing like so  "mark-task/{TodoId}"